_data/doc_version_index.yml

```diff
@@ -129,3134 +129,1257 @@ develop/index.md:
 - v2.24
 - v2.23
 develop/reference/.buildinfo:
```

Comment:
Should this line to `develop/reference` be removed as well?

Reply:
Added the auto generated file that removes all the references to develop/reference from `_data/doc_version_index.yml`. Have tested on my local; website renders perfectly fine.